src/appMenu.js

```diff
@@ -7,9 +7,12 @@ import './css/dining.css';
 import './css/news.css';
 import './css/footer.css';
 import './css/product-menu.css';
+import './css/contact.css';
 // JAVASCRIPTS!
```

Comment:
Je importeerd hier heel veel JavaScript bestanden. Maar het is beter om die functies uit die bestanden te exporteren zodat je ze in dit bestand kan gebruiken. Hier is een goede video om te zien hoe je dat doet: [https://youtu.be/NSnckA3LyGw](https://youtu.be/NSnckA3LyGw).

Het is namelijk zo dat als je functies hebt gedefinieerd in die JavaScript bestanden, maar vervolgens nergens gebruikt, kunnen ze door de "named export" gedetecteerd worden in VScode, dat zie je dan meteen. En Vite ziet dat ook, dus die zorgt dan ook dat ze niet in je build terecht komen. Het is zonde om code te publiceren die niet gebruikt word, en daar kan de gebruiker last van ondervinden als dat veel gebeurt.